frontend/language/src/nb.json

```
@@ -1017,6 +1018,17 @@
   "settings_modal.local_changes_tab_info_text": "Du kan gjøre endringer lokalt her i Altinn Studio før det lastes opp til det sentrale repositoriet. Når de er lastet opp, vil endringene bli tilgjengelig for alle. ",
   "settings_modal.open_button": "Åpne modal",
   "settings_modal.policy_tab_heading": "Tilgangsregler",
+  "settings_modal.setup_tab_heading": "Oppsett",
+  "settings_modal.setup_tab_invalid_date_or_time": "Du må velge både dato og tid.",
+  "settings_modal.setup_tab_start_before_end": "Sluttdato kan ikke være før startdato.",
+  "settings_modal.setup_tab_switch_autoDeleteOnProcessEnd": "Automatisk sletting etter innsending",
+  "settings_modal.setup_tab_switch_copyInstanceSettings_enabled": "Vis i arkiv hos bruker etter innsending",
+  "settings_modal.setup_tab_switch_messageBoxConfig_hideSettings_hideAlways": "Bruker kan kopiere instansen",
```

Jeg mistenker at verdien på de tekst-nøkklene er skal bytte plass, slik som foreslått under. Samtidig blir det riktig å si "Skjul" istedenfor "Vis" i "hideSettings_hideAlways". Fordi når den er på så vil den skjule fra akrivet.


```suggestion
  "settings_modal.setup_tab_switch_copyInstanceSettings_enabled": "Bruker kan kopiere instansen",
  "settings_modal.setup_tab_switch_messageBoxConfig_hideSettings_hideAlways": "Skjul i arkiv hos bruker etter innsending ",
```

---

Fikser!